hpi-schul-cloud/schulcloud-server
# BC-4293 - Topic does not copy with empty content Learning material

# Description
If we have a topic with the "Learning material" content which is empty, we cannot copy the topic and share topic. We have note "The copy process may take longer for large files. The content will be available shortly." topic copy does not appear. 

If I have a course with a topic in which the content of "Learning material" is empty, then the course will not copy with all topics. topics that have the content of "Learning material" empty will not be copied.

QA note:
Change tested on env:
https://nbc-bc-4293-empty-content.cd.dbildungscloud.dev/
https://brb-bc-4293-empty-content.cd.dbildungscloud.dev/
No issues found. All accepted criteria done. see movie jira tickiet. 




## Links to Tickets or other pull requests

BC-4293

## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
